config/nginx.conf

```diff
@@ -0,0 +1,27 @@
+# This is the version used in development environments
+server {
+    listen 8079;
+    root /src;
+
+    location / {
+        include uwsgi_params;
+        uwsgi_pass web:8077;
+        uwsgi_pass_request_headers on;
+        uwsgi_pass_request_body on;
+    }
+
+    location @webpack {
```

Comment:
I haven't used a named location before, but based on a quick read of the docs, it looks to me like this is not needed since it's not ref'd anywhere else (unless I missed something) and it's not used for routing otherwise.